Aptoide/aptoide-client-v8
# [Hack/AS-93] Add curation BN button

**What does this PR do?**

   Adds new bottom navigation icon to navigate to a new Curation list fragment
  Created base fragment that will be used in another ticket
  In the first commit all the BN files were moved to a specific package (just check the second commit to avoid all the file changes due to the move)
  The icon and string still need to be created and confirmed

**Database changed?**

No

**Where should the reviewer start?**

BottomNavigationActivity.java
BottomNavigationMapper.java
BottomNavigationNavigator.java

**How should this be manually tested?**

Press the new BN icon and check if it breaks (it should display an empty fragment

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/AS-93


**Code Review Checklist**
